openbas-front/src/admin/components/assets/Index.tsx

```diff
@@ -22,6 +23,8 @@ const Index = () => {
           <Route path="" element={<Navigate to="endpoints" replace={true} />} />
           <Route path="endpoints" element={errorWrapper(Endpoints)()} />
           <Route path="asset_groups" element={errorWrapper(AssetGroups)()} />
+          {/* Not found */}
+          <Route path="*" element={<NotFound/>}/>
```

Comment:
Pretty sure you do not need these because the one you put in admin/Index.tsx will trigger if no route match in this file

Reply:
Unfortunately, this redirect system is necessary everywhere.
In the case of assets, the admin/index redirects all URLs that start with assets to the assets/index file.
So, all /assets/ URLs are managed by this file, like the URL assets/toto.
So, management at this level is required.